.github/workflows/release-candidate.yml

```diff
@@ -0,0 +1,72 @@
+name: SDK Release RC
+
+on:
+  push:
+    branches:
+      - rc
```

Comment:
Its better to be `release/*` then just a single branch called `rc`

Reply:
This would overcomplicate things quite a lot in my opinion. Would require all of us to know, at the point of merging, what is the correct branch.

Curtis for example, working with anoncreds for months. Correct? He would right now have a PR ready in a branch that is not correct.

Why is that better?

Reply:
But also, you have 1 branch right, imagine, RC1.0.5, in the middle of your work in 1.0.5_rc23, you add a breaking change to the branch. The branch you were merging into, is incorrect.

We are following the recommendations, btw. Find below
https://semantic-release.gitbook.io/semantic-release/recipes/release-workflow/distribution-channels

Reply:
Having a `rc` branch means you have a long lived branch. This is not ideal. Release candidate branches should be short lived and deleted when the release is accepted and done.

Also if there is any fix the fix should be done in a PR from the main and merged to the main, then this fix commit should be cherry picked to the release branch.